common/policies.proto

```diff
@@ -69,6 +69,7 @@ message ImplicitMetaPolicy {
 // ApplicationPolicy captures the diffenrent policy types that
 // are set and evaluted at the application level.
 message ApplicationPolicy {
+    option deprecated = true;
```

Comment:
We can/should be able to remove this entirely as it's never been released, though I understand not wanting to break things in the interim.